Perlang.Tests/ConsoleApp/ProgramTest.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using Perlang.ConsoleApp;
+using Xunit;
+
+namespace Perlang.Tests.ConsoleApp
+{
+    public class ProgramTest
+    {
+        private readonly Program subject;
+        private readonly List<string> output = new List<string>();
+
+        public ProgramTest()
+        {
+            subject = new Program(standardOutputHandler: s => output.Add(s), runtimeErrorHandler: e => throw e);
+        }
```

Comment:
This class adds some rudimentary testing of the `Program`, i.e. the `perlang` entry point. We can expect more to be added in this are as we move along. :tada: